lib/style.dart

```diff
@@ -397,34 +408,68 @@ enum Display {
   INLINE,
   INLINE_BLOCK,
   LIST_ITEM,
+  NONE,
 }
 
 class FontSize {
   final double size;
+  final String units;
 
-  const FontSize(this.size);
+  const FontSize(this.size, this.units);
```

Comment:
Would have preferred to have the units as optional param, with `''` as default.

Alternatively, use named constructors for perc, px and rm?

Reply:
I should have made it optional... don't know what I was thinking I was probably a little tired 😅 I'll do the same for `LineHeight`.

Reply:
Fixed.